usaspending_api/api_contracts/contracts/v2/disaster/agency/loans.md

```diff
@@ -67,7 +67,7 @@ Returns loan spending details of Agencies receiving supplemental funding budgeta
 
 ## Filter (object)
 + `def_codes` (required, array[DEFC], fixed-type)
-+ `award_type_codes` (optional, array[string], fixed-type)
++ `award_type_codes` (optional, array[enum[string]], fixed-type)
```

Comment:
This contract has other issues, presumably because two different people worked on it.  The two different flavors of the endpoint do not accept the same sort fields.  Do we want to take least common denominator?

Reply:
It's really a lose-lose situation unless we change up the endpoint some. If we don't show `id` on the API Contract then the non-award type codes contract is wrong because the default would also be off. But if we leave it then the API Contract fails if you run it.

Reply:
Unfortunately we can't drop the `id` field because it is necessary for links to the agency page. I think the inconsistencies do need to be fixed this sprint. This ticket highlights a problem with the `id` values being returned 
https://federal-spending-transparency.atlassian.net/browse/DEV-5635

Reply:
@tony-sappe Since you will be touching the endpoints do you mind handling the sort portion of this contract not passing Dredd when you actually run it (instead of a dry-run) and I will make sure that the `award_type_codes` portion passes as part of this PR?

Reply:
I might get pulled into something else, but I will add a note to the Jira ticket for whoever does work on it